presto-product-tests/src/main/java/io/prestosql/tests/cassandra/TestSelect.java

```diff
@@ -313,6 +313,25 @@ public void testSelectClusteringMaterializedView()
         onCassandra(format("DROP MATERIALIZED VIEW IF EXISTS %s.%s", KEY_SPACE, mvName));
     }
 
+    @Test(groups = CASSANDRA)
+    public void testSelectTupleType()
+    {
+        String tableName = "insert_tuple_table";
+
+        onCassandra(format("DROP TABLE IF EXISTS %s.%s", KEY_SPACE, tableName));
+
+        onCassandra(format("CREATE TABLE %s.%s (key int, value frozen<tuple(int, int)>, PRIMARY KEY (key))",
+                 KEY_SPACE, tableName));
+
+        query(format("INSERT INTO %s.%s.%s (key, value) VALUES (1, (1,1))", CONNECTOR_NAME, KEY_SPACE, tableName));
```

Comment:
You mapped `tuple` to presto `VARCHAR`, therefore the tuple value should be `'(1,1)'`. Current `(1,1)` is identified as `ARRAY`